lametro/templates/base.html

```diff
@@ -66,6 +66,9 @@
                     <li>
                         <a href="{% url 'lametro:archive-search' %}">Archive Search</a>
                     </li>
+                    <li>
```

Comment:
Our navigation is starting to grow quite a bit, causing it to wrap to a new line even on a pretty big screen (1100+ px). Can you open an issue to consult with Metro about organizing the top nav into sub-menus, or perhaps moving some of the meta items such as About and Contact into the footer?